components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.policies/policies.js

```diff
@@ -27,6 +27,44 @@ function onRequest(context) {
             return options.fn(this);
         }
     });
-    var userModule = require("/app/modules/user.js").userModule;
-    return {permissions: userModule.getUIPermissions()};
-}
\ No newline at end of file
+    var page = {};
+    var policyModule = require("/app/modules/policy.js")["policyModule"];
+    var userModule = require("/app/modules/user.js")["userModule"];
+    var response = policyModule.getAllPolicies();
+    if (response["status"] == "success") {
+        var policyListToView = response["content"];
+        page["policyListToView"] = policyListToView;
+        var policyCount = policyListToView.length;
+        if (policyCount == 0) {
```

Comment:
Ack, Fixed in commit https://github.com/wso2/carbon-device-mgt/pull/303/commits/051493db6a3fdb2e75cc7d1f48cb3ed407622312